pkgs/development/python-modules/bootstrapped-build/default.nix

```diff
@@ -0,0 +1,105 @@
+{ stdenv
+, lib
+, python
+, requiredPythonModules
+, unzip
+, wrapPython
+
+# Bootstrapping deps
+, flit-core
+, pep517
+, tomli
+, pyparsing
+, six
+, installer
+, wheel
+, setuptools
+, packaging
+, build
+}:
+
+let
+  runtimeDeps = [
+    build
+    setuptools
+    wheel
+    flit-core
+    packaging
+    pep517
+    tomli
+    pyparsing
+    six
+    installer
+  ];
+  # runtimeDeps = lib.remove python (requiredPythonModules [ build installer setuptools wheel ]);
+  data = map (drv: lib.getAttrs ["pname" "src" ] drv) runtimeDeps;
+  # json = builtins.toFile "instructions.json" (builtins.toJSON data);
+  srcs = with lib; catAttrs "src" runtimeDeps;
+in stdenv.mkDerivation rec {
+  name = "${python.libPrefix}-bootstrapped-build-installer";
```

Comment:
I think it would be nice if we had some version attribute but honestly I have no idea what we should use because this is just a collection of files.